projects/skps/tools

```diff
@@ -0,0 +1 @@
+../../tools/
```

Comment:
Is this file necessary?

Reply:
No. My apology for that. It's a soft link to the tools folder, i just add it by mistake.